yb-voyager/cmd/exportDataStatusCommand.go

```diff
@@ -199,13 +205,16 @@ func runExportDataStatusCmd() ([]*exportTableMigStatusOutputRow, error) {
 			}
 			if exportingLeaf > 0 {
 				finalStatus = "EXPORTING"
-			} else if doneLeaf == len(snapshotStatus) {
-				finalStatus = "DONE"
 			} else if not_started == len(snapshotStatus) {
-				finalStatus = "NOT_STARTED"
+				finalStatus = "NOT-STARTED"
+			} else {
```

Comment:
I'm assuming this is just a re-order, not relevant?

Reply:
yeah mostly, just that this reorder helps now for the partition tables as we are also sending the root table's status which is `NOT-STARTED`.

Reply:
Ah okay. Pls add a comment here to explain this special case.